apps/vr-tests/src/stories/Icon.stories.tsx

```diff
@@ -3,7 +3,7 @@ import Screener from 'screener-storybook/src/screener';
 import { storiesOf } from '@storybook/react';
 import { FabricDecorator } from '../utilities/index';
 import { Icon, IconType, getIconClassName, Fabric } from '@fluentui/react';
-import * as IconNames from '../../../../packages/icons/src/IconNames';
+import * as IconNames from '../../../../packages/font-icons-mdl2/src/IconNames';
```

Comment:
Can you change this to a package name import instead? (and add this package as a dep of vr-tests if needed)